src/Config/Configuration.php

```diff
@@ -23,8 +21,8 @@ private function __construct(
     ) {
         $flagJson = json_decode($this->flagsConfig->response, true);
         $banditsJson = json_decode($this->banditsConfig?->response ?? "", true);
-        $this->flags = FlagConfigResponse::fromJson($flagJson ?? []);
-        $this->bandits = BanditParametersResponse::fromJson($banditsJson ?? []);
+        $this->flags = FlagConfigResponse::fromArray($flagJson ?? []);
```

Comment:
@aarsilv caught that `$json` was being use to improperly describe a value that was an object. in string form, we use JSON to serialize the object - once it is decoded/inflated/parsed into an object, it's no longer json - it's an array (or an Object). This kept me up at night, so here I am sneaking it into this PR

Reply:
>This kept me up at night

Oh no! Didn't mean to cause you this much trouble! 😅

Reply:
It's self inflicted, really. Once pointed out, the imprecision was deafening. IDE rename made it really easy, so no trouble at all